fitbenchmarking/fitbenchmark_one_problem.py

```diff
@@ -14,76 +14,88 @@
 from fitbenchmarking.fitting.plotting import plot_helper, plots
```

Comment:
Can't seem to add comment above but lines 7-9 are not alphabetical order

Reply:
done